src/apps/ubeswap/positions.ts

```diff
@@ -17,6 +17,7 @@ import { stakingRewardsAbi } from './abis/staking-rewards'
 import farms from './data/farms.json'
 import { NetworkId } from '../../types/networkId'
 import { getTokenId } from '../../runtime/getTokenId'
+import { getUniswapV3PositionDefinitions } from '../uniswap/positions'
```

Comment:
Initially I thought we should avoid inter-app imports like this, to make it easier to split this repo (if we ever get to this and we have so many apps it becomes necessary, or for a more permissionless version of hooks).
but I think here it's definitely the right thing to do.

Reply:
Another option would be to move shared stuff to a /shared directory or something like that, so apps can import from /shared but not other apps.